ERROR:root:<class 'Exception'>
Traceback (most recent call last):
  File "c:\Users\vikto\Documents\Old computer files\SFA\Scramble_for_Africa.py", line 10, in <module>
    setup.misc_setup(global_manager)
  File "c:\Users\vikto\Documents\Old computer files\SFA\modules\setup.py", line 251, in misc_setup
    game_transitions.set_game_mode('main_menu', global_manager)
  File "c:\Users\vikto\Documents\Old computer files\SFA\modules\game_transitions.py", line 113, in set_game_mode
    global_manager.get('sound_manager').play_random_music('main menu')
  File "c:\Users\vikto\Documents\Old computer files\SFA\modules\data_managers.py", line 897, in play_random_music
    self.music_transition(random.choice(possible_songs))
  File "c:\Users\vikto\Documents\Old computer files\SFA\modules\data_managers.py", line 857, in music_transition
    pygame.mixer.set_volume(original_volume)
AttributeError: module 'pygame.mixer' has no attribute 'set_volume'
